semver crate doesn't compile with default features disabled in 1.0.1

A simple `cargo check --no-default-features` from the 1.0.1 tag fails with:
```
   Compiling semver v1.0.1 (/home/nt/tmp2/semver)
error[E0433]: failed to resolve: use of undeclared type `Vec`
   --> src/impls.rs:152:27
    |
152 |         let comparators = Vec::from_iter(iter);
    |                           ^^^ not found in this scope
    |
help: consider importing one of these items
    |
1   | use alloc::vec::Vec;
    |
1   | use crate::Vec;
    |

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: could not compile `semver`

To learn more, run the command again with --verbose.
```
Other simple repro: `cargo init; cargo add semver --no-default-features; cargo check`

Doesn't happen on the 1.0.0 tag. This probably doesn't get used much for this crate, but we hit this on a repository where we were trying to reduce feature use all over to speed up compiles.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -19,6 +19,9 @@ jobs:
         with:
           toolchain: ${{matrix.rust}}
       - run: cargo test
+      - run: cargo check --no-default-features
+      - run: cargo check --features serde
+      - run: cargo check --no-default-features --features serde
 
   node:
     name: Node

--- a/src/impls.rs
+++ b/src/impls.rs
@@ -1,3 +1,4 @@
+use crate::backport::*;
 use crate::identifier::Identifier;
 use crate::{BuildMetadata, Comparator, Prerelease, VersionReq};
 use core::cmp::Ordering;

